ui/src/main/webapp/css/windup-web.css

```diff
@@ -112,9 +112,9 @@ td {
     font-size: 12px;
 }
 
-a[target='_blank']::before {
+.external-link::before {
```

Comment:
Ah, I can see the reason now - in context menu we want to have bigger icon :) So having it there by default is not good there..

You can ignore comment above ;)